doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -612,6 +612,12 @@ julia> 2(x-1)^2 - 3(x-1) + 1
 3
 ```
 
+!!! warning
+    Notice that, in this case, the precedence of numeric literal coefficient is
+    still the same as that of the unary operators. Hence, the expression
+    `6/2(2+1)` leads to `1` instead of the conventional result `9`, which is
```

Comment:
I'm not sure whether 9 is "the conventional result" here. I'm not immediately aware of other programming languages that allow multiplication by juxtaposition. Maybe someone else can chime in here. But in scientific literature, `6/2(2+1)` really is somewhat ambiguous - if `(6/2)*(2+1)` is meant, typically, one would either set `6/2` as a fraction or write this as `6(2+1)/2`; if that has not been done, that is somewhat indicative of the intended meaning being `6/(2(2+1))`.

I therefore also wonder whether this actually justifies a `warning` block.